benchmarks/filter_3d.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from pyinstrument import Profiler
+
+from cellfinder_core.detect.filters.volume.volume_filter import VolumeFilter
+
+# Use random data for signal data
+ball_z_size = 3
+
+
+def gen_signal_array(ny, nx):
+    shape = (ball_z_size, ny, nx)
+    return np.random.randint(low=0, high=65535, size=shape, dtype=np.uint16)
```

Comment:
I think 65535 is right - for uint16 the range is from 0 to 2**16 - 1, which in total is 2**16 possible numbers (because 0 counts as a number)

Reply:
yes, so `65535` is a value we could allow, and currently don't allow because according to [the randint docs](https://numpy.org/doc/stable/reference/random/generated/numpy.random.randint.html) high is the exclusive end or the distribution?
>high: int or array-like of ints, optional
    If provided, one above the largest (signed) integer to be drawn from the distribution (see above for behavior if high=None). If array-like, must contain integer values

Reply:
Ah my bad, I missed the "exclusive" bit, I will commit your suggestion 😄

Reply:
I definitely could have been clearer about what I was referring to in my original comment! :)